src/glmtools.jl

```diff
@@ -11,6 +11,27 @@ type LogLink <: Link end
 type ProbitLink <: Link end
 type SqrtLink <: Link end
 
+"""
+    linkfun(::Link, μ)
+
+Link function mapping μ to η (the linear predictor).    
```

Comment:
Also, `::Link` is incorrect. You could write it as `Link`, and change the description to something like:
```julia
Compute η (the linear predictor) from μ using link function of type `Link`.
```

The same pattern should be used for similar functions below.